opentech/apply/funds/utils.py

```diff
@@ -0,0 +1,42 @@
+from opentech.apply.activity.messaging import messenger, MESSAGES
+from opentech.apply.users.models import User
+
+
+def can_alter_reviewers(instance, user):
```

Comment:
For external only: `can_alter_external_reviewers` 

I should have got it right the first time :roll_eyes: